### ISSUE TITLE
Extension completions don't work when the packages are in the same file

### ISSUE THREAD

**Issue by Mee-Tree** [scalameta/metals#5241]
### Describe the bug

Define an extension method and its use case in the same file but in different packages

```scala
package a:
  extension (num: Int)
    def increment = num + 1

package b:
  @main def main: Unit =
    println(1.inc@@)
```

When typing `1.i`, `1.in`, `1.inc`, or any other prefix of `increment` there is no completion for the method.

![Peek 2023-05-18 15-42](https://github.com/scalameta/metals/assets/30271846/7d60a5e3-ed58-483c-925a-a4dc22914de8)

### Expected behavior

Increment method should be suggested the same way it does with objects or when the packages are in different files.

![image](https://github.com/scalameta/metals/assets/30271846/ddae3e0a-efb5-4fc6-bc86-7314c50ef76a)

### Operating system

Linux

### Editor/Extension

VS Code

### Version of Metals

v0.11.12

### Extra context or search terms

- extension method
- first mentioned in [#5235](https://github.com/scalameta/metals/pull/5235#issuecomment-1551814713)

**Comment by tgodzik** [comment]
Thanks for reporting and a reproduction!

**Comment by tgodzik** [comment]
Closed in #5260 

Let us know if you still see an issue!
